README/Data/Nat.agda

```diff
@@ -12,31 +12,34 @@ module README.Data.Nat where
 -- The natural numbers and various arithmetic operations are defined
 -- in Data.Nat.
 
-open import Data.Nat
+open import Data.Nat using (ℕ; _+_; _*_)
 
+-- _*_ has precedence 7 over precedence 6 of _+_
+-- precedence of both defined in module Agda.Builtin.Nat
 ex₁ : ℕ
-ex₁ = 1 + 3
+ex₁ = 1 + 3 * 4
 
 -- Propositional equality and some related properties can be found
 -- in Relation.Binary.PropositionalEquality.
 
-open import Relation.Binary.PropositionalEquality
+open import Relation.Binary.PropositionalEquality using (_≡_; refl)
 
 ex₂ : 3 + 5 ≡ 2 * 4
 ex₂ = refl
 
 -- Data.Nat.Properties contains a number of properties about natural
 -- numbers.
 
-import Data.Nat.Properties as Nat
+import Data.Nat.Properties as Nat using (*-comm; +-identityʳ)
 
 ex₃ : ∀ m n → m * n ≡ n * m
 ex₃ m n = Nat.*-comm m n
```

Comment:
Should we remove the `Nat.` qualification here then?

Reply:
Yes, because of redundancy. I did not realize.

Reply:
Okay, once you've removed it I'm happy and I'll merge this in.

Reply:
OK, done!